docs/modules/server.rst

```diff
@@ -21,6 +21,10 @@ For example, a client request could be {‘endpoint’: ‘get_guild_count’, 
 This JSON is processed upon a request being made, and checks for a registered route matching the name of the endpoint supplied.
 It then calls the method linked to said route and returns the payload to the client.
 
-.. automodule:: discord.ext.ipc.server
+.. currentmodule:: discord.ext.ipc.server
+
+.. py:function:: route
+
+.. autoclass:: Server
     :members:
     :undoc-members:
```

Comment:
This can be removed with the new content additions.

Reply:
wdym

Reply:
<sub>This is why showing surrounding lines was a bad idea, GitHub.</sub>

```suggestion

```

Reply:
> This is why showing surrounding lines was a bad idea, GitHub.

Still lost, but ill remove `:undoc-members:` since Logan *should* have the docstrings done for all relevant methods so there is no need to show ones without docstrings

Reply:
That's the idea!

Reply:
Ah easy